priv/repo/migrations/20160503010933_copy_states_csv.exs

```diff
@@ -0,0 +1,46 @@
+defmodule HospitalReadmissionServer.Repo.Migrations.CopyStatesCsv do
+  use Ecto.Migration
+
+  def change do
+    abs_path = Path.absname('/usr/share/Reformatted_Census_Data.csv')
+    IO.puts(abs_path)
+
+    # Remove not null restriction on inserted_at and updated_at
+    execute """
+      ALTER TABLE states
+      ALTER COLUMN inserted_at
+      DROP NOT NULL,
+      ALTER COLUMN updated_at
+      DROP NOT NULL;
+    """
+
+    execute """
+      COPY states(
+        name,
+        population,
+        abbrev
+      )
+      FROM '#{abs_path}'
+      WITH CSV HEADER;
+    """
+
+    # elixir writes timestamps without timezone  = postgres.localtimestamp
+    execute """
+      UPDATE hospitals
```

Comment:
Forgot to change this to states, that's why our states inserted at/updated at was null.